### ISSUE TITLE
Reconnecting to server leaves shutdown-dialog open

### ISSUE THREAD

**Issue by emente** [foosel/OctoPrint#1959]
#### What were you doing?

1.Shutdown Machine that runs OctoPrint via System->Shutdown

Machine shuts down, Shutdown alert in Browser still open
The dialog that confirms a lost data connection to server appears

2.Click reconnect to get UI to initiate server reconnect

UI unghosts and is fully functional, but Shutdown alert still visible.

#### What did you expect to happen?
Modal dialogs disappear after successful reconnect || Modal dialogs disappear after button click

#### What happened instead?
(Already clicked) Shutdown alert still visible, cancel button functional

#### Branch & Commit or Version of OctoPrint
master branch/1.3.4

**Comment by GitIssueBot** [comment]
Hi @emente,

It looks like there is some **information missing** from your bug report that will be needed in order to solve the problem. Read the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md) which will provide you with a template to fill out here so that your bug report is ready to be investigated (I promise I'll go away then too!).

If you did not intend to report a bug but wanted to **request a feature or brain storm** about some kind of development, please take special note of the title format to use as described in the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md).

**Please do not abuse the bug tracker as a support forum** - if you have a question or otherwise need some kind of help or support refer to the [Mailinglist](https://groups.google.com/group/octoprint) or the [G+ Community](https://plus.google.com/communities/102771308349328485741) instead of here.

Also **make sure you are at the right place** - this is the bug tracker of the official version of OctoPrint, not the Raspberry Pi image OctoPi nor any unbundled third party OctoPrint plugins or unofficial versions. Make sure too that you have **read through the [Frequently Asked Questions](https://github.com/foosel/OctoPrint/wiki/FAQ)** and searched the [**existing tickets**](https://github.com/foosel/OctoPrint/search?q=&ref=cmdform&type=Issues) for your problem - try multiple search terms please.

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next two weeks (until 2017-06-20 18:30 UTC) I'll close this ticket so it doesn't clutter the bug tracker. This is nothing personal, so please just be considerate and help the maintainers solve this problem quickly by following the guidelines linked above. Remember, the less time the devs have to spend running after information on tickets, the more time they have to actually solve problems and add awesome new features. Thank you!

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being, so don't expect any replies from me :) Your ticket is read by humans too, I'm just not one of them.

**Comment by foosel** [comment]
I cannot reproduce what I think you are describing, but I'm unsure if I'm looking at the right thing. Could you provide a screenshot or better yet a video showing the behaviour?

**Comment by emente** [comment]
Just tried to reproduce the bug and could only after tens of tries. Only thing i can think of is that Raspbian shuts down so very fast that the octoprint process is terminated before the shutdown-dialog is hidden.

As the severity of this bug is very low and it's reproducibility seem to depend on the moon phase... I'm okay with WONTFIX.

**Comment by foosel** [comment]
Hm, the hiding of the shutdown dialog happens in the browser without interaction from the server, so how fast the backend shuts down shouldn't influence this.

But if it's tricky to reproduce it definitely sounds like a possible race condition in the frontend code. I'll leave this open and keep an eye out for it.

**Comment by emente** [comment]
At first i thought it could have something to do with the fact that i installed OctoPrint on top of a ROS fuerte image (because the Pi+Image are leftovers from my robot who got an waaay faster intel cpu). But last week the Pi died, with a new one (but same sd card) the behaviour is now as expected. 

Thanks for taking time.

Another bug ticket done for ♬

**Comment by foosel** [comment]
Closing this since I haven't heard from anyone else running into this for over two years.
